graylog2-shared/src/main/java/org/graylog2/shared/security/ldap/LdapEntry.java

```diff
@@ -53,4 +73,13 @@ public String getEmail() {
         }
         return email;
     }
+
+    @Override
+    public String toString() {
+        return "LdapEntry{" +
```

Comment:
Just a remark: we could use [`MoreObjects.ToStringHelper`](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/MoreObjects.ToStringHelper.html) here.